app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -532,6 +538,15 @@ class MessagingManager @Inject constructor(
                                 sendNotification(jsonData)
                             }
                         }
+                        COMMAND_FLASHLIGHT -> {
+                            val command = jsonData[NotificationData.COMMAND]
+                            if (command in DeviceCommandData.ENABLE_COMMANDS) {
```

Comment:
if this command requires a certain android version we should check for it here like we do with `command_bluetooth`

Reply:
Done! ✅

I also [updated the docs PR](https://github.com/home-assistant/companion.home-assistant/pull/1131/commits/0b1e9999597c4f77da2d9cd0d4cc542e84e3940a).